pkgs/servers/apache-flume/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, jre, makeWrapper, majorVersion ? "1.9.0" }:
+
+let
+  versionMap = {
+    "1.9.0" = {
+      flumeVersion = "1.9.0";
+      sha256 = "069nnakibv4pdra06fnq08nb7qypzx8j586r4fmw8kflpxdfswq3";
+    };
+  };
+in
+
+with versionMap.${majorVersion};
+
+stdenv.mkDerivation rec {
+  version = "${flumeVersion}";
+  name = "apache-flume-${version}";
```

```suggestion
  pname = "apache-flume";
```

`pname` appends the version name on its own.